test/model.jl

```diff
@@ -106,6 +109,70 @@ function test_model()
             err = ErrorException("Constraints of type MathOptInterface.ScalarAffineFunction{Float64}-in-MathOptInterface.Interval{Float64} are not supported by the solver.")
             @test_throws err @constraint model 0 <= x + 1 <= 1
         end
+
+        @testset "Add bridge" begin
+            function mock()
+                mock = MOIU.MockOptimizer(JuMP.JuMPMOIModel{Float64}(),
+                                          eval_variable_constraint_dual=false)
+                optimize!(mock) = MOIU.mock_optimize!(mock, [1.0],
+                        (MOI.SingleVariable, MOI.GreaterThan{Float64}) => [2.0])
+                MOIU.set_mock_optimize!(mock, optimize!)
+                return mock
+            end
+            factory = with_optimizer(mock)
+            @testset "before loading the constraint to the optimizer" begin
+                @testset "with_optimizer at Model" begin
+                    model = Model(factory)
+                    @variable(model, x)
+                    JuMP.add_bridge(model, NonnegativeBridge)
+                    c = @constraint(model, x in Nonnegative())
+                    JuMP.optimize!(model)
+                    @test JuMP.value(x) == 1.0
+                    #@test JuMP.value(c) == 1.0
```

Comment:
Why commented? (Same below)

Reply:
I was waiting for https://github.com/JuliaOpt/JuMP.jl/pull/1728 to be merged